mobilizon_reshare/event/event.py

```diff
@@ -51,17 +53,25 @@ def _fill_template(self, pattern: Template) -> str:
     def format(self, pattern: Template) -> str:
         return self._fill_template(pattern)
 
-    def to_model(self) -> Event:
-        return Event(
-            name=self.name,
-            description=self.description,
-            mobilizon_id=self.mobilizon_id,
-            mobilizon_link=self.mobilizon_link,
-            thumbnail_link=self.thumbnail_link,
-            location=self.location,
-            begin_datetime=self.begin_datetime.astimezone(self.begin_datetime.tzinfo),
-            end_datetime=self.end_datetime.astimezone(self.end_datetime.tzinfo),
-        )
+    def to_model(self, db_id: Optional[UUID] = None) -> Event:
+        kwargs = {
+            "name": self.name,
+            "description": self.description,
+            "mobilizon_id": self.mobilizon_id,
+            "mobilizon_link": self.mobilizon_link,
+            "thumbnail_link": self.thumbnail_link,
+            "location": self.location,
+            "begin_datetime": self.begin_datetime.astimezone(
+                self.begin_datetime.tzinfo
+            ),
+            "end_datetime": self.end_datetime.astimezone(self.end_datetime.tzinfo),
+            "last_update_time": self.last_update_time.astimezone(
+                self.last_update_time.tzinfo
+            ),
+        }
+        if db_id is not None:
```

Comment:
questa cosa non ha molto senso farla qui. Non è un concern di questa classe gestirsi il db_id. Siccome questo metodo ritorna un dizionario, nulla ti vieta di fare questa cosa altrove.

Reply:
questo metodo ritorna un modello `Event`. Se l'id non è concern di questa classe allora sto metodo non dovrebbe proprio essere qui imho, ma probabilmente in `storage.query` o simili. Idem per `MobilizonEvent.from_model`, cosi togliamo tutte le dipendenze verso i moduli dei modelli, o non ci ho capito nulla?

Reply:
in effetti sì, sarebbe più coerente. Vedi se non è troppa sbatta. Perché passare un valore per metterlo dentro un dizionario così fa un po' ridere

Reply:
ora dovrebbe essere ok